src/transit_equity/census/income.py

```diff
@@ -38,10 +38,16 @@ def min_income(self):
     def max_income(self):
         return self._max_income
 
-# B19001 - HOUSEHOLD INCOME IN THE PAST 12 MONTHS
-# The min_income and max_income have been assigned as -1 for the total field, 
-# # since they are not used in the calculation of average household income
 class INCOME_DISTRIBUTION_COLUMNS(Enum):
+    """
+    This Enum class contains the details of the income distribution columns in the census data
+        that will be relevant for the low income analysis
+
+    Table: B19001 - HOUSEHOLD INCOME IN THE PAST 12 MONTHS
+
+    The min_income and max_income have been assigned as -1 for the total field, 
+        since they are not used in the calculation of average household income
+    """
```

Comment:
I am not sure about giving the '200000 and more' max of 200000. What is the purpose of making it?

Reply:
I am adding math.inf as the max_income. Its not like we will be using these columns to calculate average or anything for now, since there are other tables for dedicated to those purposes. 
If we do, we will change it accordingly.

Reply:
Done